src/components/Card/Card.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { CardDiv, ImageDiv, InfoDiv, Title, Description } from './Card.styled';
+
+const Card = ({ title, description, link }) => {
+  return (
+    <>
```

Comment:
This fragment isn't necessary because `CardDiv` already wraps around everything.